models/work-item-link-type.go

```diff
@@ -0,0 +1,276 @@
+package models
+
+import (
+	"github.com/almighty/almighty-core/app"
+	convert "github.com/almighty/almighty-core/convert"
+	"github.com/almighty/almighty-core/gormsupport"
+	satoriuuid "github.com/satori/go.uuid"
+)
+
+const (
+	TopologyNetwork         = "network"
+	TopologyDirectedNetwork = "directed_network"
+	TopologyDependency      = "dependency"
+	TopologyTree            = "tree"
+
+	SystemWorkItemLinkTypeBugBlocker = "system.bug-blocker"
```

Comment:
But this the following line is also defined in `models/workitemtype.go`, so why not define the link type here?

```go
SystemBug              = "system.bug"
```